lib/tasks/cleanup.rake

```diff
@@ -18,4 +18,17 @@ namespace :cleanup do
       Cart.destroy_all_instrument_only_carts(5.days.ago)
     end
   end
+
+  namespace :price_groups do
+    desc "remove cancer center"
+    task :cancer_center, [:cancer_center_name] => :environment do |_t, args|
+      cc_name = args[:cancer_center_name] || "Cancer Center Rate"
+      pg = PriceGroup.find_by(name: cc_name)
+      abort("No cancer center price group found with name #{cc_name}") unless pg
```

Comment:
We could do a `find_by!` and let that exception take care of this for us. This could be better UI, though.